src/generator.js

```diff
@@ -423,6 +423,16 @@ ${this.documentation.renderLinksInText(member.discouraged)}
   generateDoc(name, outName) {
     const content = fs.readFileSync(path.join(this.srcDir, name)).toString();
     let nodes = this.filterForLanguage(md.parse(content));
+    if (outName === 'release-notes.mdx') {
```

Comment:
- How does this work in `/next`? We should keep all versions there.
- I guess we'll have to re-publish after each language release? Perhaps it's better to not worry about this?

Reply:
let's wait for now and maybe not land the language binding release notes when we release JS.